server/src/main/java/org/jboss/as/server/moduleservice/ServiceModuleLoader.java

```diff
@@ -128,9 +130,20 @@ protected Module preloadModule(final String name) throws ModuleLoadException {
         }
     }
 
+    /**
+     * @deprecated Use {@link ServiceModuleLoader#findModule(String)}
```

Comment:
Please change this to:

@deprecated Will be made protected in line with this method in the parent class

There may have been a reason for this being public in 2011 when it was changed to public, but I don't see it now. It should be an internal call within ModuleLoader, not used by outside code.  (WF and WF Core code don't call this method directly.)

I doubt any code is calling this and if they are they are misusing something. Removing ModuleIdentifier will be a breaking change to whatever the code might be so they might as well stop doing whatever that is.

We'll be removing this method later